lib/ansible/galaxy/data/default/collection/galaxy.yml.j2

```diff
@@ -0,0 +1,66 @@
+### REQUIRED
+
+# this can be a company/brand/organization or product namespace
+# under which all content lives
+namespace: {{ namespace }}
+
+
+# the designation of this specific collection
+name: {{ collection_name }}
+
+
+# semantic versioning compliant version designation
+version: 1.0.0
+
+# the filename for the readme file which can be either markdown (.md) or
```

Comment:
```suggestion
# the filename for the readme file in markdown (.md)
```

Reply:
most recent meeting between docs and galaxy folks are limiting this to .md for now as that is compatible with ansible-docs output and .rst is not.

Reply:
Thanks for letting me know, will fix up the skeleton.